src/SprykerEco/Service/AkeneoPim/AkeneoPimServiceInterface.php

```diff
@@ -537,4 +539,401 @@ public function getProductMediaFilesListPerPage($limit = 10, $withCount = false,
+     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourcePageInterface
+     */
+    public function getAssetCategoriesListPerPage(int $limit = 10, bool $withCount = false, array $queryParameters = []): AkeneoResourcePageInterface;
+
+    /**
+     * Specification:
+     * - Gets a single reference entity.
+     *
+     * @api
+     *
+     * @param string $code
+     *
+     * @return array
+     */
+    public function getReferenceEntity(string $code): array;
+
+    /**
+     * Specification:
+     * - Gets a cursor to iterate over the list of reference entities.
+     *
+     * @api
+     *
+     * @param array $queryParameters
+     *
+     * @return \Akeneo\Pim\ApiClient\Pagination\ResourceCursorInterface
+     */
+    public function getAllReferenceEntities(array $queryParameters = []): ResourceCursorInterface;
+
+    /**
+     * Specification:
+     * - Gets a single reference entity record.
+     *
+     * @api
+     *
+     * @param string $referenceEntityCode
+     * @param string $recordCode
+     *
+     * @return array
+     */
+    public function getReferenceEntityRecord(string $referenceEntityCode, string $recordCode): array;
+
+    /**
+     * Specification:
+     * - Gets a cursor to iterate over the list of records of a given reference entity.
+     *
+     * @api
+     *
+     * @param string $referenceEntityCode
+     * @param array $queryParameters
+     *
+     * @return \Akeneo\Pim\ApiClient\Pagination\ResourceCursorInterface
+     */
+    public function getReferenceEntityRecords(string $referenceEntityCode, array $queryParameters = []): ResourceCursorInterface;
+
+    /**
+     * Specification:
+     * - Gets a single reference entity attribute.
+     *
+     * @api
+     *
+     * @param string $referenceEntityCode
+     * @param string $attributeCode
+     *
+     * @return array
+     */
+    public function getReferenceEntityAttribute(string $referenceEntityCode, string $attributeCode): array;
+
+    /**
+     * Specification:
+     * - Gets the list of the attributes of a given reference entity.
+     *
+     * @api
+     *
+     * @param string $referenceEntityCode
+     * @param array $queryParameters
+     *
+     * @return \Akeneo\Pim\ApiClient\Pagination\ResourceCursorInterface
+     */
+    public function getReferenceEntityAttributes(string $referenceEntityCode, array $queryParameters = []): ResourceCursorInterface;
+
+    /**
+     * Specification:
+     * - Get an attribute option for a given attribute of a given reference entity.
+     *
+     * @param string $referenceEntityCode
+     * @param string $attributeCode
+     * @param string $attributeOptionCode
+     *
+     * @return array
+     */
+    public function getReferenceEntityAttributeOption(string $referenceEntityCode, string $attributeCode, string $attributeOptionCode): array;
+
+    /**
+     * Specification:
+     * - Get the list of attribute options of a given attribute for a given reference entity.
```

Comment:
Please check specifications. Somewhere it `Get`, somewhere it `Gets`

Reply:
Only two "Get" there :) Done